README.md

```diff
@@ -206,6 +206,7 @@ Resource name is written in the plural form.
 |---------|-----------|-------|-------|
 |`stream=`|Return JSON in streaming format|`false`|`true`|
 |`q[field_name]=`|A unique query parameter for each field for filtering|(empty)|`q[id]=1,2,5`, `q[admin]=true&q[registered]=true`|
+|`sort=`|Sort parameters|`false`|`id, -age, -created_at`|
```

Comment:
- Default value should be (empty)
- Please explain the correspondences of (none)/`-` prefixes and `asc`/`desc` somewhere. 
- Example shoule be "`id`, `-age`, `id,-created_at`" (imo)